lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl

```diff
@@ -1408,64 +1408,67 @@ POST http://smd:27779/hsm/v2/Inventory/RedfishEndpoints
 HTTP 201
 Content-Type: application/json
 [Asserts]
-jsonpath "$[0].URI" == "/hsm/v2/Inventory/RedfishEndpoints/x0c0s3b0"
+jsonpath "$[0].URI" == "/hsm/v2/Inventory/RedfishEndpoints/x0c1s3b0"
 
+#
 # Get RedfishEndpoints we just added.
-GET http://smd:27779/hsm/v2/Inventory/RedfishEndpoints
```

Comment:
Getting rid of the base cases?

Reply:
Ah, I presume you mean checking `v2/Inventory/RedfishEndpoints` as well as `v2/Inventory/RedfishEndpoints/{xname}`. That is a good point, and I will add that as a test case.

Reply:
Added in 861e634.

Reply:
For RedfishEndpoints, Components, and ComponentEndpoints.